uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -387,6 +394,48 @@ public void setMqttTimeoutSec(int mqttTimeoutSec) {
         log.info("MQTT timeout set to {} second(s)", mqttTimeoutSec);
     }
 
+    /**
+     * Sets MQTT content type to transmit.
+     *
+     * @param contentType MQTT content type to transmit
+     */
+    @And("I set MQTT 'content type' to {string} to transmit")
+    public void setMqttTxContentType(String contentType) {
+        this.txContentType = contentType;
+        log.info("MQTT content type set to {} to transmit", contentType);
+    }
+
+    /**
+     * Sets MQTT content type to receive.
+     *
+     * @param contentType MQTT content type to receive
+     */
+    @And("I set MQTT 'content type' to {string} to receive")
+    public void setMqttRxContentType(String contentType) {
+        this.rxContentType = contentType;
+        log.info("MQTT content type set to {} to receive", contentType);
+    }
+
+    /**
+     * Clear MQTT content type to transmit.
+     */
+    @And("I clear MQTT 'content type' to transmit")
```

Comment:
Reorder code to  be updates txContentType together.
Rename the step.
Other steps has no 'to transmit' sentence in name

Reply:
The step was renamed in 41edf65c896419d5e3fc6a8684a8ffd6126319b9